README.md

```diff
@@ -1,14 +1,73 @@
 # How to Add the Syncfusion Blazor Dropdown Menu Component to a Blazor Server App   
 
-The Blazor Dropdown Menu allows users to toggle between the different contextual overlay that displays the list of action items. In this video, you will learn how to add a simple Blazor Dropdown Menu to a Blazor server app. You will see how to add icons and change their position. You will also learn how to navigate to other webpages as well as display the Dropdown Menu items horizontally. Finally, you will see how to hide the Dropdown Menu arrow and bind a click event to the component.
- 
-Example: https://blazor.syncfusion.com/demos/buttons/dropdown-menu
-
-Documentation: https://blazor.syncfusion.com/documentation/drop-down-menu/getting-started
+[Blazor Dropdown Menu](https://www.syncfusion.com/blazor-components/blazor-dropdown-menu?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples) is a graphical user interface component that lets users choose from a list of actions that can be triggered. Allows users to toggle between the different contextual overlay that displays the list of action items. In this video, you will learn how to add a simple Blazor Dropdown Menu to a Blazor server app. You will see how to add icons and change their position. You will also learn how to navigate to other webpages as well as display the Dropdown Menu items horizontally. Finally, you will see how to hide the Dropdown Menu arrow and bind a click event to the component.
```

Comment:
check the UTM parameter in all places?
![image](https://user-images.githubusercontent.com/61148953/161229830-39b34701-7764-4645-8dca-76ccbdd2705f.png)

Reply:
Resolved